torchbenchmark/models/drq/drq.py

```diff
@@ -43,7 +43,7 @@ def forward_conv(self, obs):
             conv = torch.relu(self.convs[i](conv))
             self.outputs['conv%s' % (i + 1)] = conv
 
-        h = conv.view(conv.size(0), -1)
+        h = conv.reshape(conv.size(0), -1)
```

Comment:
Can you add one-line comment mention that this is an improvement that doesn't exist in the upstream code (with link to the upstream code at https://github.com/denisyarats/drq/blob/master/drq.py#L48)?

Reply:
@xuzhao9 We can submit a PR to upstream too. Does it sound good to you?

Reply:
> @xuzhao9 We can submit a PR to upstream too. Does it sound good to you?

Sure, if you can submit a PR to upstream and get accepted, we don't need the comment line.

Reply:
I've added one-line comment as suggested for now.
I'll submit a PR to upstream this. If it is accepted, let me submit a following-up PR later to this benchmark repo to remove the one-line comment. Is it okay?

Reply:
Yes, that shoulds perfect to me!

Reply:
FYI, I've submitted a PR to the upstream drq repo: https://github.com/denisyarats/drq/pull/27.